### PR TITLE
Adds a GitHub Action to validate RSIs

### PR DESCRIPTION
**Pull request by ike709**
## About the PR 
Adds a GitHub action to check RSI `meta.json` files (when edited/added in a PR) and see if they have, among other things, a `copyright` field and an SS14-compatible `license`.

**Changelog**

N/A, not player-facing.

### CODE REVIEW

File: Resources/Textures/Mobs/Aliens/Xenos/xeno.rsi/meta.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "version": 1,
-  "license": "CC-BY-SA-3.0",
+  "license": "deez nuts",
```
Comment (DrSmugleaf) [inline Resources/Textures/Mobs/Aliens/Xenos/xeno.rsi/meta.json]: sir

this is an arby's

**Comment by ike709** [comment]
Check the commits to see what happens if an RSI is invalid.

**Comment by ike709** [comment]
Accepted licenses:
`"CC-BY-SA-3.0",
            "CC-BY-SA-4.0",
            "CC-BY-NC-SA-3.0",
            "CC-BY-NC-SA-4.0",
            "CC0-1.0"`

**Comment by ike709** [comment]
Derived from swept's `meta.json` file.

**Comment by SweptWasTaken** [comment]
Very

File: .github/workflows/validate-rsis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: RSI Validator
+
+on: [pull_request]
```
Comment (PaulRitter) [inline .github/workflows/validate-rsis.yml:3]: run it on push too

**Review by PaulRitter — CHANGES_REQUESTED** [review]

**Comment by PaulRitter** [comment]
we lintin now bois

File: .github/workflows/validate-rsis.yml (hunk 2 above)
Comment (ike709) [inline .github/workflows/validate-rsis.yml:3]:
```suggestion
on: [push, pull_request]
```

**Pull request merged**